package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@microsoft.azure/autorest.testserver",
-  "version": "3.0.3",
+  "version": "3.0.4",
```

Comment:
Probably update that to 3.0.5 when you other PR merge

Reply:
yeah, i'm going to merge this one first, i'm leaving the other one on the back-burner for a bit, need to make some style changes to my autorest python pr for that, so I'm going to leave this one 3.04. thanks @timotheeguerin